### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by aleksandr-mazur**
https://aleksandr-mazur.github.io/layout_html-form/
https://aleksandr-mazur.github.io/layout_html-form/report/html_report/

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,11 @@
...
+  display: block;
+}
+
+.input:not(:last-child) {
+  margin-bottom: 10px;
+}
+
+fieldset {
+  margin-bottom: 20px;
+}
```
Comment (stanislav-topikha) [inline src/style.css:11]: [no tag styling ](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md#:~:text=%5BCODE%20STYLE%5D%20%2D%20Don%27t%20style%20tags%2C%20use%20class%20selector%20instead)

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]
it shouldn't work this way
yes or not, not both
Screenshot 2022-08-15 at 12 52 32

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,138 @@
...
+
+      <fieldset class="field">
+        <legend>Registration</legend>
+
+        <label class="input">
+          E-mail:
+          <input type="email"
+           name="email" 
+          placeholder ="email@example.com"
+          >
```
Comment (valerii-derkach) [inline src/index.html:67]:
```suggestion
          E-mail:
          <input
            type="email"
            name="email" 
            placeholder ="email@example.com"
          >
```

File: src/index.html [hunk 3]
```diff
@@ -11,7 +11,138 @@
...
+        </label>
+        
+        <label class="input">
+          Password:
+          <input type="password"
+           name="password"
+           minlength="8"
+          >
```
Comment (valerii-derkach) [inline src/index.html:75]:
```suggestion
          <input
            type="password"
            name="password"
            minlength="8"
          >
```

File: src/index.html [hunk 4]
```diff
@@ -11,7 +11,138 @@
...
+        Do you love cats?
+          <input type="radio"
+         name="cats" 
+         value="yes">
+        Yes
+          <input type="radio"
+         name="cats"
+         value="no">
+        No
+        </label>
```
Comment (valerii-derkach) [inline src/index.html:92]:
```suggestion
      <label class="input">
          Do you love cats?
          <label>
            <input
              type="radio"
              name="cats"
              value="yes"
            >
            Yes
          </label>

          <label>
            <input
              type="radio"
              name="cats"
              value="no"
            >
            No
          </label>
        </label>
```
image

File: src/index.html [hunk 5]
```diff
@@ -11,7 +11,138 @@
...
+
+        <label class="input">
+          What is your favorite color?
+          <input type="color"
+           name="color">
+        </label>
```
Comment (valerii-derkach) [inline src/index.html:98]: same here

File: src/index.html [hunk 6]
```diff
@@ -11,7 +11,138 @@
...
+
+        <label class="input">
+          What time do you go to bed?
+          <input type="time"
+           name="time"
+           step="2"
+          >
```
Comment (valerii-derkach) [inline src/index.html:105]: same here

**Review by valerii-derkach — CHANGES_REQUESTED** [review]
please fix identantions
[checklist ](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)2,3

**Review by liliia-livandovskaya — COMMENTED** [review]
Still no fix on this. There should be yes or no
Знімок екрана 2022-08-25 о 15 42 10

File: src/index.html [hunk 7]
```diff
@@ -11,7 +11,155 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
```
Comment (liliia-livandovskaya) [inline src/index.html:14]: remove comments

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
GJ! Make final fixes

**Pull request closed without merging**